debian/rules

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with python2
```

Comment:
Use **pybuild**, check [here](https://wiki.debian.org/Python/LibraryStyleGuide?action=show&redirect=Python#debian.2Frules)

Reply:
Also, make sure the tests pass both in Python2 & 3